### PR TITLE
HomeScreen/CardsCarousel

### PR DESCRIPTION
**Pull request by sksinha2410**
Created `CardsCarousel.tsx` and `DebitCard.tsx`
  - It shows `BankName` , `Balance` , `CardHolderName` , `CardNumber`.
  - Added option to `hide/unhide` Balance.

## Sample Code to used:
```
import React from 'react';
import { SafeAreaView, StatusBar } from 'react-native';
import CardsCarousel from './src/UI/Components/CardsCarousel/CardsCarousel';
import styles from './src/UI/Components/DebitCard/Style';

const sampleData = [
  { id: 1, bankName: 'Axis Bank', balance: '₹9244.56', cardNumber: '1234 5678 9101 1121', cardHolder: 'kshitiz Jha' },
  { id: 2, bankName: 'State Bank of India', balance: '₹4321.87', cardNumber: '4321 8765 1011 2112', cardHolder: 'Shivam Sinha' },
  { id: 3, bankName: 'HDFC Bank', balance: '₹5678.90', cardNumber: '5678 1234 9101 1121', cardHolder: 'Prem Kumar' },
];

const App = () => {
  return (
    <SafeAreaView style={styles.appContainer}>
      <StatusBar barStyle="dark-content" />
      <CardsCarousel data={sampleData} />
    </SafeAreaView>
  );
};

export default App;
```

## Output 
Screenshot_20240701-115055
Screenshot_20240701-115057
Screenshot_20240701-115059

### CODE REVIEW

File: src/UI/Components/CardsCarousel/Pagination.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { View, StyleSheet, Animated, Dimensions } from 'react-native';
+
+const { width } = Dimensions.get('window');
+
+const Pagination = ({ data, scrollX }: { data: any[], scrollX: Animated.Value}) => {
```
Comment (Jha2508) [inline src/UI/Components/CardsCarousel/Pagination.tsx]: use types for Props

Comment (Jha2508) [reply src/UI/Components/CardsCarousel/Pagination.tsx]: documentation missing

File: src/UI/Components/CardsCarousel/Pagination.tsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+              },
+            ]}
+          />
+        );
+      })}
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```
Comment (Jha2508) [inline src/UI/Components/CardsCarousel/Pagination.tsx]: move styles to different file

File: src/UI/Components/CardsCarousel/DebitCard.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { View, Text, TouchableOpacity, Image, ImageBackground } from 'react-native';
+import styles from './styles';
```
Comment (Jha2508) [inline src/UI/Components/CardsCarousel/DebitCard.tsx]: why are we keeping all styles at same place? move it to a separate style for dashboard

File: src/UI/Components/CardsCarousel/DebitCard.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+type DebitCardProps = {
+  item: {
+    bankName: string;
+    balance: string;
+    cardNumber: string;
+    cardHolder: string;
+  };
+};
+
+const DebitCard = ({ item }: DebitCardProps) => {
```
Comment (Jha2508) [inline src/UI/Components/CardsCarousel/DebitCard.tsx]:
```suggestion
const DebitCard = (props : DebitCardProps) => {
```

**Comment by Jha2508** [comment]
Why folder structure is not followed here for components?
Please make sure the folder structure is always followed

**Comment by sksinha2410** [comment]
All comments has been addressed

**Review by Jha2508 — APPROVED** [review]

**Comment by Jha2508** [comment]
Change file naming structure here. index.tsx and styles.ts should be there for each component

**Pull request merged**